data-sample/semopenalex-data-sample.trig

```diff
@@ -0,0 +1,2494 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix ns1: <https://semopenalex.org/property/> .
```

Comment:
instead of rdflib generated namespace, should we use namespace with `sp` for semopenalex property?

Reply:
I have followed the format and naming of the .trig files we provide at https://semopenalex.s3.amazonaws.com/browse.html . Since they are generated with rdflib too I reused the rdflib generated namespaces. If users use the data sample for testing their scripts I would like to keep the data sample similar to the real data.